strum_macros/src/helpers/case_style.rs

```diff
@@ -1,5 +1,9 @@
 use heck::{CamelCase, KebabCase, MixedCase, ShoutySnakeCase, SnakeCase, TitleCase};
-use syn::Ident;
+use std::convert::TryFrom;
```

Comment:
Unfortunately, `TryFrom` isn't available until `1.34`. It annoys me, but I'm not so annoyed that I'm willing to up the MSRV yet.

Reply:
Haha, hadn't looked at the MSRV. I'll just switch from `TryFrom` implementation to constructor fn, does that sound good?

Reply:
Actually, can make it a `FromStr` implementation!